deeplearning4j/deeplearning4j-core/src/test/java/org/deeplearning4j/regressiontest/RegressionTest100b6.java

```diff
@@ -0,0 +1,390 @@
+/*
```

Comment:
You've still got the old regression test commit in this PR too (from another separate PR). This commit/file shouldn't be here. If you don't know how to fix this - ask.